commit 41c5580f955acd28ea8c83313458fb45e5cdb776
Author: Mike Manning <mmanning@brocade.com>
Date:   Wed Mar 1 09:55:28 2017 +0000

    net: bridge: allow IPv6 when multicast flood is disabled
    
    [ Upstream commit 8953de2f02ad7b15e4964c82f9afd60f128e4e98 ]
    
    Even with multicast flooding turned off, IPv6 ND should still work so
    that IPv6 connectivity is provided. Allow this by continuing to flood
    multicast traffic originated by us.
    
    Fixes: b6cb5ac8331b ("net: bridge: add per-port multicast flood flag")
    Cc: Nikolay Aleksandrov <nikolay@cumulusnetworks.com>
    Signed-off-by: Mike Manning <mmanning@brocade.com>
    Acked-by: Nikolay Aleksandrov <nikolay@cumulusnetworks.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1385_41c5580f955acd28ea8c83313458fb45e5cdb776.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1385_41c5580f955acd28ea8c83313458fb45e5cdb776.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
